packages/connectivity/README.md

```diff
@@ -0,0 +1,27 @@
+# connectivity_tizen
+
+The Tizen implementation of [`connectivity`](https://github.com/flutter/plugins/tree/master/packages/connectivity).
+
+## Usage
+
+This package is not an _endorsed_ implementation of `connectivity`. Therefore, you have to include `connectivity_tizen` alongside `connectivity` as dependencies in your `pubspec.yaml` file.
+
+```yaml
+dependencies:
+  connectivity: ^0.4.9
+  connectivity_tizen: ^1.0.0
+```
+
+Then you can import `connectivity` in your Dart code:
+
+```dart
+import 'package:connectivity/connectivity.dart';
+```
+
+For detailed usage, see https://github.com/flutter/plugins/tree/master/packages/connectivity/connectivity#usage.
+
+## Supported devices
+
+This plugin is supported on these types of devices:
+
+- Galaxy Watch (running Tizen 4.0 or later)
```

Comment:
Please specify if any privilege is required to use this plugin in README. There's an example in the checklist or https://github.com/flutter-tizen/plugins/tree/master/packages/path_provider#required-privileges.

Reply:
Done.